modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+// * microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-dry-run-v2_{context}"]
+= Performing a dry run for V2
+
+You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored, as well as any images that would be pruned from the mirror registry. A dry run also allows you to catch any errors with your image set configuration early or use the generated list of images with other tools to carry out the mirroring operation. 
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plugin.
+* You have created the image set configuration file.
+
+.Procedure
+
+. Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \// <1>
+  docker://registry.example:5000            \// <2>
+  --dry-run                                  <3>
+----
+<1> Pass in the image set configuration file that was created. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Specify the mirror registry. Nothing is mirrored to this registry as long as you use the `--dry-run` flag.
+<3> Use the `--dry-run` flag to generate the dry run artifacts and not an actual image set file.
++
+.Example output
+[source,terminal]
+----
+Checking push permissions for registry.example:5000
+Creating directory: oc-mirror-workspace/src/publish
+Creating directory: oc-mirror-workspace/src/v2
+Creating directory: oc-mirror-workspace/src/charts
+Creating directory: oc-mirror-workspace/src/release-signatures
+No metadata detected, creating new workspace
+wrote mirroring manifests to oc-mirror-workspace/operators.1658342351/manifests-redhat-operator-index
+
+...
+
+info: Planning completed in 31.48s
+info: Dry run complete
+Writing image mapping to oc-mirror-workspace/mapping.txt
+----
+
+. Navigate into the workspace directory that was generated:
++
+[source,terminal]
+----
+$ cd oc-mirror-workspace/
+----
+
+. Review the `mapping.txt` file that was generated.
++
+This file contains a list of all images that would be mirrored.
+
+. Review the `pruning-plan.json` file that was generated.
++
+This file contains a list of all images that would be pruned from the mirror registry when the image set is published.
++
+[NOTE]
+====
+The `pruning-plan.json` file is only generated if your oc-mirror command points to your mirror registry and there are images to be pruned.
```

Comment:
as kasturri mentioned pruning-plan.json is not generated in v2

Reply:
Done